cmd/containerboot/main.go

```diff
@@ -837,6 +873,55 @@ func installEgressForwardingRule(ctx context.Context, dstStr string, tsIPs []net
 	return nil
 }
 
+// natLocalTrafficToDst installs DNAT and SNAT rules to ensure that traffic is
+// forwarded to the new destination.
+func forwardNonTailscaleTrafficToDst(ctx context.Context, local, dst string, nfr linuxfw.NetfilterRunner) error {
```

Comment:
This is almost identical to `installEgressForwardingRule` - do we need two different functions for this, or would one be sufficient?